modules/client/src/internal/constants.ts

```diff
@@ -33,3 +46,36 @@ export const UNAVAILABLE_DAO_METADATA: DaoMetadata = {
   description: "(the DAO metadata is not available)",
   links: [],
 };
+export const UNSUPPORTED_RELEASE_METADATA_LINK: PluginRepoReleaseMetadata = {
+  name: "(unsupported metadata link)",
+  description: "(the metadata link is not supported)",
+  images: {},
+};
+export const EMPTY_RELEASE_METADATA_LINK: PluginRepoReleaseMetadata = {
+  name: "(the release has no metadata)",
+  description: "(the release did not define any content)",
+  images: {},
+};
+
+export const UNAVAILABLE_RELEASE_METADATA: PluginRepoReleaseMetadata = {
+  name: "(unavailable metadata)",
+  description: "(the release metadata is not available)",
+  images: {},
+};
+
+export const UNSUPPORTED_BUILD_METADATA_LINK: PluginRepoBuildMetadata = {
+  ui: "(unsupported metadata link)",
```

Comment:
What does `ui` mean in the context of a plugin build?

Reply:
I suppose It refers to the url to the url where the ui of the plugin is stored but I'm not sure, I followed the metadata schema of the plugins that are deployed atm